content/docs/addons.md

```diff
@@ -21,8 +21,13 @@ The add-ons below are in the development (unminified) version of React only:
 
 The add-ons below are considered legacy and their use is discouraged. They will keep working in observable future, but there is no further development.
 
+<<<<<<< HEAD
+- [`PureRenderMixin`](pure-render-mixin.html). Use [`React.PureComponent`](/docs/react-api.html#reactpurecomponent) instead.
+- [`shallowCompare`](shallow-compare.html), a helper function that performs a shallow comparison for props and state in a component to decide if a component should update. We recommend using [`React.PureComponent`](/docs/react-api.html#reactpurecomponent) instead.
+=======
 - [`PureRenderMixin`](pure-render-mixin.html). Use [`React.PureComponent`](/docs/react-api.html#reactpurecomponent) instead.
```

Comment:
Looks like this didn't get merged correct

Reply:
Sorry, I'm so silly.